app/resources/locale/en/en.po

```diff
@@ -559,139 +569,133 @@ msgid "Hexadecimal"
 msgid "Input port name:"
-msgstr ""
+msgstr "Input port name:"
 
 #: app/scripts/services/forms.js:1565
-#, fuzzy
 msgid "Input ports"
-msgstr "Enter the input ports"
+msgstr "Input ports"
 
-#: app/views/menu.html:635
-#, fuzzy
+#: app/views/menu.html:653
 msgid "Install (Stable)"
-msgstr "Install"
+msgstr "Install (Stable)"
 
-#: app/views/menu.html:656
+#: app/views/menu.html:674
 msgid "Install Development version"
-msgstr ""
+msgstr "Install Development version"
 
-#: app/scripts/services/tools.js:1118
+#: app/scripts/services/tools.js:1229
 msgid ""
 "Install the <b>STABLE Toolchain</b>. This operation requires Internet "
 "connection <br><p><b>NOTE:</b> You need to disconnect your VPN (if any) to "
 "allow the toolchain installation</p> <p>Do you want to continue?</p>"
 msgstr ""
+"Install the <b>STABLE Toolchain</b>. This operation requires Internet "
+"connection <br><p><b>NOTE:</b> You need to disconnect your VPN (if any) to "
+"allow the toolchain installation</p> <p>Do you want to continue?</p>"
 
-#: app/scripts/services/tools.js:1144
-#, fuzzy
+#: app/scripts/services/tools.js:1255
 msgid ""
 "Install the DEVELOPMENT toolchain. It will be downloaded. This operation "
 "requires Internet connection. Do you want to continue?"
 msgstr ""
-"The toolchain will be updated. This operation requires Internet connection. "
-"Do you want to continue?"
+"Install the DEVELOPMENT toolchain. It will be downloaded. This operation "
+"requires Internet connection. Do you want to continue?"
 
-#: app/scripts/services/tools.js:1166
-#, fuzzy
+#: app/scripts/services/tools.js:1277
 msgid ""
 "Install the LATEST STABLE toolchain. It will be downloaded. This operation "
 "requires Internet connection. Do you want to continue?"
 msgstr ""
-"The toolchain will be updated. This operation requires Internet connection. "
-"Do you want to continue?"
+"Install the LATEST STABLE toolchain. It will be downloaded. This operation "
+"requires Internet connection. Do you want to continue?"
 
-#: app/scripts/services/tools.js:1496
+#: app/scripts/services/tools.js:1607
 msgid "Installation completed"
 msgstr "Installation completed"
 
-#: app/scripts/services/tools.js:1274
-#, fuzzy
+#: app/scripts/services/tools.js:1385
 msgid "Installing"
-msgstr "Install"
+msgstr "Installing"
 
-#: app/scripts/services/drivers.js:421 app/scripts/services/tools.js:1362
+#: app/scripts/services/drivers.js:447 app/scripts/services/tools.js:1473
 msgid "Internet connection required"
 msgstr "Internet connection required"
 
-#: app/scripts/services/graph.js:290
+#: app/scripts/services/graph.js:298
 msgid "Invalid <i>Pull up</i> connection:<br>block already connected"
 msgstr "Invalid <i>Pull up</i> connection:<br>block already connected"
 
-#: app/scripts/services/graph.js:304
+#: app/scripts/services/graph.js:312
 msgid "Invalid <i>Pull up</i> connection:<br>only <i>Input</i> blocks allowed"
 msgstr "Invalid <i>Pull up</i> connection:<br>only <i>Input</i> blocks allowed"
```

Comment:
I think globally 'pull up' could be 'pullup', what do you think?

Reply:
I prefer pull-up (Arduino influence here). Actually as I'm writing "pullup" github is trying to correct me saying to use "pull-up" or "pull up" 🤦🏽‍♂️
Tricky word, very context dependent...